halconfig/settings.js

```diff
@@ -105,7 +105,7 @@ window.spinnakerSettings = {
   authEnabled: authEnabled,
   authTtl: 600000,
   gitSources: ['stash', 'github', 'bitbucket'],
-  triggerTypes: ['git', 'pipeline', 'docker', 'cron', 'jenkins'],
+  triggerTypes: ['git', 'pipeline', 'docker', 'cron', 'jenkins', 'travis'],
```

Comment:
Could you please make a corresponding update to the settings.js files under the https://github.com/spinnaker/spinnaker repo?